README.md

```diff
@@ -1,24 +1,95 @@
 # `Gjallarhorn`
 
-Gjallarhorn is a tool for Brawlhalla community tournament runners to generate
-properly formatted JSON for Twitch overlays. This supports overlays for Game,
-Players, Queue, Ticker, Casters, and Lower Thirds. It pulls data from start.gg
-as well as Brawlhalla's Stats API.
+Gjallarhorn is a tool for Brawlhalla community tournament organizers to generate properly formatted JSON for Twitch overlays. Gjallarhorn can generate JSON for multiple use cases, including tickers, casters, lower thirds, queues, players, and games. It pulls data from start.gg as well as Brawlhalla's Stats API.
 
-## Usage
+## Getting Started
+
+You will need a start.gg API key to use Gjallarhorn. If you don't have one already, get one at https://developer.start.gg/docs/authentication.
 
-To start it up, you can run the following commands:
+To start Gjallarhorn, run the following commands in the project:
 
 ```
 npm run build
-node . -s [startgg-key-here]
+node . -s [your-startgg-key-here]
 ```
 
-Check out https://developer.start.gg/docs/authentication to get started with an
-API key if you don't have one already. You can fill out the cards with the
-proper details you want on the overlay, and hit "Push". You will find the output
-files served over http at http://localhost:3000/api/json or the raw json files
-in `packages/cli/output`
+Then, launch the dashboard at http://localhost:3000.
+
+## Usage
+
+You will find 6 different cards on the dashboard: Casters, Game, Ticker, Players, Queue, and Lower Thirds. You can fill out the cards with the details you want on the overlay and hit "Push". You will find the output files served over http at http://localhost:3000/api/json or the raw JSON files in `packages/cli/output`.
```

Comment:
Either the names of each card here should link to that respective section the documentation below (like [this](https://github.com/giovcandido/sedona?tab=readme-ov-file#contents)), or we should put the sections for the cards in expandable sections (like [this](https://github.com/AntonioFalcaoJr/EventualShop?tab=readme-ov-file#mag_right-research))

Reply:
EXPANDABLE SECTIONS??? when did they update markdown

Reply:
I did the former, but I'm leaving this as unresolved for the time being to see if I can't figure out expandable sections.

Reply:
The doc is in expandable sections now. I left the Project Overview and Contributors mostly untouched.